docs/core/tools/dotnet-publish.md

```diff
@@ -43,7 +43,9 @@ The `dotnet publish` command's output is ready for deployment to a hosting syste
 
 ### MSBuild
 
-The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. Any parameters passed to `dotnet publish` are passed to MSBuild. The `-c` and `-o` parameters map to MSBuild's `Configuration` and `PublishDir` properties, respectively.
+The `dotnet publish` command calls MSBuild, which invokes the `Publish` target. If the `IsPublishable` property in a project's *.csproj* file is set to `false`, the `Publish` target can't be invoked, and in that case the `dotnet publish` command doesn't do anything with the project.
```

Comment:
> doesn't do anything with the project

Does it raise an error in such a case?

Reply:
No error because this is expected behavior. If you have a solution with two projects, one of which is not publishable, when you run `dotnet publish` on the solution you get a successful run that publishes only the publishable project.